LICENCE-COMMERCIAL

```diff
@@ -0,0 +1,31 @@
+Commercial License for Positron LT
+
+Copyright (c) 2023 Rene Nagel
+
+This Commercial License (the "License") grants [Company Name], a corporation organized and existing under the laws of its jurisdiction ("Licensee"), the right to use, modify, and distribute Positron LT (the "Design"), a 3D printer design and associated documentation, for commercial purposes, subject to the following terms and conditions:
+
+1. Fees and Payments:
+   Licensee agrees to pay a royalty fee equal to 5% of the gross revenue directly attributable to the use of the Design in Licensee's products or services. Payments shall be made quarterly within 30 days after the end of each calendar quarter.
```

Comment:
I'm not so sure about writing the exact percantage in the license.
@Audiotronix decides

Reply:
I get your point, but also it might be better being clear and upfront.
Its hidden away in the license file anyway, no one apart from the companies is gonna look at that. So very little change of getting a bad rep from it.